`make install` should not use `go install`

Even though it is simple to use `go install`, it requires the person building the Node from scratch to set the GOBIN for this to work reliably, thus to build the Node locally, one has to be aware of Go's semantics. 

I propose to avoid this by making `make install` not to rely on `go install`, but to `cp` the results from `make build` to `/usr/local/bin` which may require the user's `sudo` permissions. Furthermore, we should use built-in GNU `install` instead of `cp`.

Is this the reason why I'm getting this error
`--> Installing Celestia
build github.com/celestiaorg/celestia-node/cmd/celestia: cannot load io/fs: malformed module path "io/fs": missing dot in first path element
make: *** [Makefile:20: install] Error 1
`

--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,9 @@
 SHELL=/usr/bin/env bash
 PROJECTNAME=$(shell basename "$(PWD)")
 LDFLAGS="-X 'main.buildTime=$(shell date)' -X 'main.lastCommit=$(shell git rev-parse HEAD)' -X 'main.semanticVersion=$(shell git describe --tags --dirty=-dev)'"
+ifeq (${PREFIX},)
+	PREFIX := /usr/local
+endif
 
 ## help: Get more info on make commands.
 help: Makefile
@@ -19,11 +22,17 @@ clean:
 	@echo "--> Cleaning up ./build"
 	@rm -rf build/*
 
-## install: Build and install the celestia-node binary into the GOBIN directory.
-install:
+## install: Build and install the celestia-node binary into the $PREFIX (/usr/local/ by default) directory.
+install: build
+	@echo "--> Installing Celestia"
+	@install -v ./build/* -t ${PREFIX}/bin/
+.PHONY: install
+
+## go-install: Build and install the celestia-node binary into the GOBIN directory.
+go-install:
 	@echo "--> Installing Celestia"
 	@go install -ldflags ${LDFLAGS}  ./cmd/celestia
-.PHONY: install
+.PHONY: go-install
 
 ## shed: Build cel-shed binary.
 cel-shed:

--- a/README.md
+++ b/README.md
@@ -33,7 +33,8 @@ See the official docs page for system requirements per node type:
 ```sh
 git clone https://github.com/celestiaorg/celestia-node.git 
 cd celestia-node
-make install
+make build
+sudo make install
 ```
 
 For more information on setting up a node and the hardware requirements needed, go visit our docs at <https://docs.celestia.org>.

